compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -2316,7 +2317,7 @@ object messages {
         }
         i"super[$staticSuperName].$memberName"
       }
-      hl"""$base cannot be defined due to a conflict between its parents when
+      em"""$base cannot be defined due to a conflict between its parents when
```

Comment:
The superCall, resolvedSuperCall and staticSuperCall defined above should be wrapped in hl in their definitions to compensate for this change.